crowbar_framework/app/assets/stylesheets/content/footer.css.scss

```diff
@@ -1,6 +1,8 @@
 footer {
+  margin-top: 40px;
+  font-size: 12px;
   text-align: center;
-  padding: 0 20px 20px 20px;
+  color: #999;
```

Comment:
Color literals like `#999` should only be used in variable declarations; they should be referred to via variable everywhere else.